### ISSUE TITLE
Default cost for panic parsing should be in dictionaries

### ISSUE THREAD

**Issue by linas** [opencog/link-grammar#390]
When panic parsing kicks in, a different acceptable "max-cost" gets used.  It's currently defaulted to 2.7 in the code `api.c:  po->disjunct_cost = 2.7;` but this value should be encoded into, and fetched from the dictionary, instead.

This is easier said than done: its still plausible that users might want to over-ride this value when experimenting, and so there is a tricky question as to whether this should also be a parse option, or not.   The simplest thing to do would be to get rid of the parse option, I think ...

This is very low priority, since nothing currently depends on this.

**Comment by ampli** [comment]
I looks like the concept of panic mode parsing belongs only to the user's application (like `link-parser`) and not at all to the LG library, and that the library only supplies the timeout facility.
So the `po->disjunct_cost = 2.7` is `api.c` is for regular (non-panic) mode parsing. `link-parser` uses 4.0 for panic mode parsing.

Did I miss something?

In PR #388 you said:
>In the long run, the default disjunct cost should be encoded in the dictionary.

I think this is a good idea, because this value depends on the design of the dictionary.
Maybe two values can be provided, the second one is to be used **by the application** for setting the disjunct-cost parse-option for panic mode parsing. It is extremely trivial to implement that.

The disjunct_cost for panic mode parsing should not be implemented as its own parse option since it is not directly used by the library. It can be implemented like getting the dictionary version. It will return an array of 2 floating-point values.

BTW, there is a comment rot in api.c:
```
                                        ...  cost 3.0, which
	 * is used only during panic-parsing.
```

**Comment by linas** [comment]
Yep. This is both easy and low priority, and basically won't matter until autogenned dictionaries, and arguably won't matter much then, either. Not sure why I opened this issue.

**Comment by ampli** [comment]
Included in issue #785.
